tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md

```diff
@@ -369,7 +369,7 @@ As you can see, the properties `BusinessPartner` and `LastName` will be marked a
 
 [ACCORDION-BEGIN [Step 9: ](Systems.json and credentials)]
 
-If you run your application on SAP Cloud Platform, the SDK can simply read the ERP destinations from the destination service. However, since the tests should run locally, we need a way to supply our tests with an ERP destination.
+If you run your application on the BTP, the SDK can simply read the ERP destinations from the destination service. However, since the tests should run locally, we need a way to supply our tests with an ERP destination.
```

Comment:
Please don't say "we" unless you are referring to SAP.

Reply:
We are planning to drop our tutorials for the Neo environment. Therefore, I wont update this any more.